Formula/uriparser.rb

```diff
@@ -4,15 +4,8 @@ class Uriparser < Formula
   head "https://github.com/uriparser/uriparser.git"
 
   stable do
```

Comment:
Please remove the stable block if no patches are present

Reply:
👍